get-tom-observations/README.md

```diff
@@ -0,0 +1,12 @@
+Get TOM Observations
+====================
+
+This folder defines a [Google Cloud Function](https://cloud.google.com/functions/).
+
+This function will query a list of observation requests from the PANOPTES Observation Portal. 
+
+This Cloud Function is called periodically by the PANOPTES units.
```

Comment:
Let's remove the last line here (it's not called periodically) and include a sentence about what TOM is as well as a link to the relevant TOM pages.